ocp_install_env.sh

```diff
@@ -276,6 +276,21 @@ sshKey: |
 fips: ${FIPS_MODE:-false}
 EOF
 
+  if [[ ! -z "$INSTALLER_PROXY" && "$IP_STACK" = "v4" ]]; then
+
+    EXT_SUBNET=${EXTERNAL_SUBNET_V6}
+    if [[ "$IP_STACK" = "v4" ]]; then
+      EXT_SUBNET=${EXTERNAL_SUBNET_V4}
+    fi
+
+    cat >> "${outdir}/install-config.yaml" << EOF
+proxy:
+  httpProxy: http://${PROVISIONING_HOST_EXTERNAL_IP}:${INSTALLER_PROXY_PORT}
+  httpsProxy: http://${PROVISIONING_HOST_EXTERNAL_IP}:${INSTALLER_PROXY_PORT}
+  noProxy: ${PROVISIONING_NETWORK},${EXT_SUBNET}
```

Comment:
If we calculate `NO_PROXY` in this file instead of `06` we could reuse it here?

Reply:
Maybe I could also centralize also all of them, will try it out

Reply:
PS: Just noticed a problem (a spurios check for ipv4 that should have been removed), fixing it